packages/data-access/src/data-access.ts

```diff
@@ -79,6 +81,8 @@ export default class DataAccess implements DataAccessTypes.IDataAccess {
    */
   private logger: LogTypes.ILogger;
 
+  private buffer: Buffer;
+
```

Comment:
(TODO for later) The consturctor should have additional arguments:
```suggestion
// Existing
synchronizationIntervalTime: DEFAULT_INTERVAL_TIME,
// New, optional, could be 0 if there is no max batch size
maxBatchSize: int,
// New, mandatory to give a period for batching
batchingIntervalTime: int,
```